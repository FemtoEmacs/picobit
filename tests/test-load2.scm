(define (foo x)
  (display x))
