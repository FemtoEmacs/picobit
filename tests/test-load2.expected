:0080000080
:05800000FBD70000D8D1
:00000001FF
